samples/nrf9160/download/src/main.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <string.h>
+#include <zephyr.h>
+#include <stdlib.h>
+#include <modem/bsdlib.h>
+#include <modem/lte_lc.h>
+#include <modem/at_cmd.h>
+#include <modem/at_notif.h>
+#include <modem/modem_key_mgmt.h>
+#include <net/download_client.h>
+
+#define URL CONFIG_SAMPLE_FILE_URL
+#define SEC_TAG CONFIG_SAMPLE_SEC_TAG
+
+#define PROGRESS_WIDTH 50
+
+#if CONFIG_SAMPLE_SECURE_SOCKET
+static const char cert[] = {
+	#include CONFIG_SAMPLE_CERT_FILE
+};
+BUILD_ASSERT(sizeof(cert) < KB(4), "Certificate too large");
+#endif
+
+static struct download_client downloader;
+static struct download_client_cfg config = {
+#if CONFIG_SAMPLE_SECURE_SOCKET
+	.sec_tag = SEC_TAG,
+#else
+	.sec_tag = -1,
+#endif
+};
+
+static s64_t ref_time;
```

Comment:
We should use `int64_t`.

Reply:
Right, I update the lib but not the sample :) thanks